src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -2040,72 +1941,6 @@ public void T007_PackageProperties()
         }
 
         [Fact]
-        public void T006_CreateExternalRelationshipInvalidUri()
```

Comment:
I'll verify when I get a chance, but I believe it turned out the failing line was 2085: `Uri uri = new Uri("a b c d e", UriKind.Absolute);`, which meant we weren't testing anything in packaging behavior.